torchvision/models/efficientnet.py

```diff
@@ -178,12 +256,19 @@ def __init__(
             raise ValueError("The inverted_residual_setting should not be empty")
         elif not (
             isinstance(inverted_residual_setting, Sequence)
-            and all([isinstance(s, MBConvConfig) for s in inverted_residual_setting])
+            and all([isinstance(s, _MBConvConfig) for s in inverted_residual_setting])
         ):
             raise TypeError("The inverted_residual_setting should be List[MBConvConfig]")
 
-        if block is None:
-            block = MBConv
+        if "block" in kwargs:
+            warnings.warn(
+                "The parameter 'block' is deprecated since 0.13 and will be removed 0.15. "
+                "Please pass this information on 'MBConvConfig.block' instead."
+            )
+            if kwargs["block"] is not None:
+                for s in inverted_residual_setting:
+                    if isinstance(s, MBConvConfig):
```

Comment:
Nit: according to the signature this should always be True, unless the caller passes the wrong type, in which case I think it is ok to break.

Reply:
Good catch. The signature was wrong. It should have been `Union[MBConvConfig, FusedMBConvConfig]`. I'll correct now.